File: GTTS/INL.MessagingService/Models/NotificationPersonsVettingVettingTypeCreated_Item.cs
using System;
using System.Collections.Generic;

namespace INL.MessagingService.Models
{
	public class NotificationPersonsVettingVettingTypeCreated_Item : INotificationPersonsVettingVettingTypeCreated_Item
	{
		public long VettingBatchID { get; set; }
		public string GTTSTrackingNumber { get; set; }
		public int VettingTypeID { get; set; }
		public string VettingType { get; set; }		

		public string Name { get; set; }
		public int OrganizerAppUserID { get; set; }
		public DateTime EventStartDate { get; set; }
		public DateTime EventEndDate { get; set; }

		public int ParticipantsCount { get; set; }
		public int CourtesyCheckTimeFrame { get; set; }

		public List<dynamic> CourtesyVetters { get; set; }

		public string RedirectorURL { get; set; }

		public string EventStart {
			get {
				return String.Format("{0:MM/dd/yyyy}", this.EventStartDate);
			}
		}
		public string EventEnd
		{
			get
			{
				return String.Format("{0:MM/dd/yyyy}", this.EventEndDate);
			}
		}		
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/notification-persons-vetting-vetting-type-created_item.ts


import { INotificationPersonsVettingVettingTypeCreated_Item } from './inotification-persons-vetting-vetting-type-created_item';

export class NotificationPersonsVettingVettingTypeCreated_Item implements INotificationPersonsVettingVettingTypeCreated_Item {
  
	public VettingBatchID: number = 0;
	public GTTSTrackingNumber: string = "";
	public VettingTypeID: number = 0;
	public VettingType: string = "";
	public Name: string = "";
	public OrganizerAppUserID: number = 0;
	public EventStartDate: Date = new Date(0);
	public EventEndDate: Date = new Date(0);
	public ParticipantsCount: number = 0;
	public CourtesyCheckTimeFrame: number = 0;
	public CourtesyVetters?: any[];
	public RedirectorURL: string = "";
	public EventStart: string = "";
	public EventEnd: string = "";
  
}


